rasa/cli/export.py

```diff
@@ -0,0 +1,217 @@
+import argparse
+import logging
+from typing import List, Text, Optional
+
+import rasa.cli.utils as cli_utils
+import rasa.core.utils as rasa_core_utils
+from rasa.cli.arguments import export as arguments
+from rasa.core.brokers.broker import EventBroker
```

Comment:
let's avoid importing all this stuff unless it's actually needed. Cause currently we load all this stuff whenever somebody does `rasa --help`